doc/docs/C++_Tutorial.md

```diff
@@ -60,7 +60,7 @@ double eps(const vec &p) {
 }
 ```
 
-which returns the dielectric function $\varepsilon(\mathbf{x})$ which is just a 2$\times$3 rectangle of $\varepsilon$=12 in the upper-left corner. Unlike in the Scheme interface, by default the origin of the coordinate system is at the *corner* of the computational cell.
```

Comment:
Here is another case where you used unicode in a LaTeX equation; it isn't safe to rely on this rendering correctly because it is not allowed in "standard" LaTeX.